MDN annos: Show less-than-2/all-engines indicators

sideshowbarker opened the pull request:
This change adds indicator symbols and text in the MDN annotations, to give information about the number of browser engines which support the feature associated with the annotation.

domenic commented:
Can we fix the FIXMEs to make reviewing this a bit easier? It seems we could merge the resources.whatwg.org PR ahead of time with no negative impact.

sideshowbarker commented:
> Can we fix the FIXMEs to make reviewing this a bit easier? It seems we could merge the resources.whatwg.org PR ahead of time with no negative impact.

Sure, yes, I’ll do that

sideshowbarker commented:
> Can we fix the FIXMEs to make reviewing this a bit easier? It seems we could merge the resources.whatwg.org PR ahead of time with no negative impact.

OK, I removed the inline styles and moved them to https://github.com/whatwg/whatwg.org/blob/sideshowbarker/mdn-annos-style-updates/resources.whatwg.org/standard-shared-with-dev.css

https://github.com/whatwg/whatwg.org/pull/308
